server/src/routes/api/v1/reviewsRouter.js

```diff
@@ -17,7 +18,7 @@ reviewsRouter.delete("/:id", async (req, res) => {
         await Vote.query().deleteById(vote.id)
       }
       const deleted = await Review.query().deleteById(reviewId)
-      return res.status(200).json()
+      return res.status(200).json({ deletedReview: deleted })
```

Comment:
`delete` returns the number of records deleted I believe. I wouldnt send back `deleted`. Just send back a success message